src/handlers/comment/handlers/assign.ts

```diff
@@ -101,17 +107,26 @@ export const assign = async (body: string) => {
     const recipient = await getWalletAddress(payload.sender.login);
     if (!recipient) {
       //no wallet found
-      commit_msg =
-        commit_msg +
-        "\n\n" +
-        "It looks like you haven't set your wallet address,\n" +
-        "please use `/wallet 0x4FDE...BA18` to do so.\n" +
-        "(It's required to be paid for the bounty)";
+      wallet_msg = "Please set your wallet address to use `/wallet 0x4FDE...BA18`";
     } else {
       //wallet found
-      commit_msg = commit_msg + "\n\n" + "Your currently set address is:\n" + recipient + "\n" + "please use `/wallet 0x4FDE...BA18` if you want to update it.";
+      wallet_msg = recipient;
+    }
+    const multiplier = await getWalletMultiplier(payload.sender.login);
+    if (!multiplier) {
+      multiplier_msg = "1.00";
+    } else {
+      multiplier_msg = multiplier.toFixed(2);
+    }
+    const issueDetailed = bountyInfo(issue);
+    if (!issueDetailed.priceLabel) {
+      bounty_msg = `Permit generation skipped since price label is not set`;
+    } else {
+      bounty_msg = (+issueDetailed.priceLabel!.substring(7, issueDetailed.priceLabel!.length - 4) * multiplier).toString() + " USD";
     }
-    return commit_msg;
+    const reason = await getWalletReason(payload.sender.login);
+    reason_msg = reason ? reason : "";
```

Comment:
you can make it simple
```suggestion
    reason_msg = reason ?? "";
```